app/src/main/res/layout/fragment_saved_login_site_info.xml

```diff
@@ -0,0 +1,159 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- This Source Code Form is subject to the terms of the Mozilla Public
+   - License, v. 2.0. If a copy of the MPL was not distributed with this
+   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:layout_marginTop="4dp"
+    android:layout_marginBottom="4dp"
+    android:orientation="vertical">
+
+    <androidx.constraintlayout.widget.ConstraintLayout
```

Comment:
Please correct me if I'm wrong, but it seems like we should be able to accomplish this with one top level `ConstraintLayout`, right?  CLs are _crazy_ slow compared to other layouts, they only improve perf because nesting is also painfully inefficient in Android.  Nesting them is kind of taking the worst of both worlds.

Reply:
U are super right here. Fixing this up now